buildSrc/build.gradle

```diff
@@ -0,0 +1,22 @@
+
+
+plugins {
+    // Support convention plugins written in Groovy. Convention plugins are build scripts in 'src/main' that automatically become available as plugins in the main build.
+    id 'groovy-gradle-plugin'
+}
+
+repositories {
+    // Use the plugin portal to apply community plugins in convention plugins.
+    gradlePluginPortal()
+}
+
+wrapper {
+    gradleVersion = '7.2'
```

Comment:
Wrapper here specified…I suspect we only want this in one place, also 7.3.3 is released